xwiki-platform-core/xwiki-platform-index/xwiki-platform-index-test/xwiki-platform-index-test-docker/src/test/it/org/xwiki/index/test/ui/DeletedAttachmentsIT.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.index.test.ui;
+
+import java.io.ByteArrayInputStream;
+import java.util.Calendar;
+import java.util.Date;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.xwiki.index.test.po.DeletedAttachmentsPage;
+import org.xwiki.test.docker.junit5.TestReference;
+import org.xwiki.test.docker.junit5.UITest;
+import org.xwiki.test.ui.TestUtils;
+import org.xwiki.test.ui.po.AttachmentsPane;
+import org.xwiki.test.ui.po.LiveTableElement;
+import org.xwiki.test.ui.po.ViewPage;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * Tests for the DeletedAttachments page.
+ *
+ * @since 12.2RC1
+ */
+@UITest
+public class DeletedAttachmentsIT
+{
+    private static String FIRST_LIVETABLE_COLUMN_ID = "xwiki-livetable-attachmentTrash-filter-1";
+
+    private static String THIRD_LIVETABLE_COLUMN_ID = "xwiki-livetable-attachmentTrash-filter-3";
+
+    @BeforeEach
+    public void setUp(TestUtils setup, TestReference testReference) throws Exception
```

Comment:
Better with `public` that's the new norm in Junit5. Allows to make sure we don't use the wrong `Test` annotation too  (the one from junit4).

Reply:
@vmassol means **without** public.

Reply:
oops yes, "without" sorry :)